spec/models/service_template_transformation_plan_task_spec.rb

```diff
@@ -514,10 +515,10 @@
               :vm_name             => src_vm_1.name,
               :transport_method    => 'vddk',
               :vmware_fingerprint  => '01:23:45:67:89:ab:cd:ef:01:23:45:67:89:ab:cd:ef:01:23:45:67',
-              :vmware_uri          => "esx://esx_user@10.0.0.1/?no_verify=1",
+              :vmware_uri          => "esx://esx_user@127.0.0.1/?no_verify=1",
```

Comment:
Why was this changed?

Reply:
@bdunne you recommended not using `:ipaddress` because it was being sequenced.  `127.0.0.1` is the default value used in the factory